docs/_docs/Users/build.md

```diff
@@ -5,6 +5,95 @@ order: 1
 toc: true
 ---
 
+## Ubuntu 18.04 and Linux Mint 19
```

Comment:
In general: can you follow the description for 16.04 more closely (just to have a common "style")? Especially: use the same steps/wording if possible and use code blocks.

Reply:
I have also tested with Ubuntu 18.04.1 and it still needs all this work.

Reply:
Ok. Anyways, I've opened an issue to figure out why this is necessary. Until that, the current description containing all steps is merged to master.